lang/po/Firmware_no.po

```diff
@@ -2528,6 +2528,26 @@ msgstr "UKJENT FEIL"
 msgid "Unexpected error occurred."
 msgstr "Det oppstod en uventet feil."
 
+#. MSG_BTN_EJECT c=8
+#: ../../Firmware/mmu2/errors_list.h:357 ../../Firmware/mmu2/errors_list.h:369
+msgid "Eject"
+msgstr "Løs ut"
+
+#. MSG_TITLE_FILAMENT_CHANGE c=20
+#: ../../Firmware/mmu2/errors_list.h:188 ../../Firmware/mmu2/errors_list.h:235
+msgid "FILAMENT CHANGE"
+msgstr "ENDRING AV FILAMENT"
+
+#. MSG_BTN_LOAD c=8
+#: ../../Firmware/mmu2/errors_list.h:356 ../../Firmware/mmu2/errors_list.h:368
+msgid "Load"
+msgstr "Laste"
```

Comment:
Would "Last inn" fit? I think that would be a little bit better.

Reply:
Personally both "Last" and "Last-Inn" works for me, but only Last is technically correct, however ambiguous without a context.

Reply:
@trondkla Thanks as the limit is 8 characters it should fit
```
01234567
Eject
Løs ut

01234567
Load
Last-inn
```

Reply:
@trondkla Do you know by chance bit Swedish as we didn't get an answer yet from the SV translation owner

Reply:
I understand and can read swedish, but I do not speak fluent.
It looks good based on the other Swedish translations.